### ISSUE TITLE
Blurry images on CIFAR10 Dataset

### ISSUE THREAD

**Issue by Milkigit** [eugenet12/pytorch-rbm-autoencoder#1]
Hi, thanks a lot for the fantastic job and informative demo.
After I tried the demo on the CIFAR10 dataset, I found the reconstructed images are quite blurry. Would you please provide some guidelines on what parameters I should adjust that can make them more clear? Really appreciated. :)

Here are the reconstructed images on CIFAR10 dataset.
Original
![image](https://user-images.githubusercontent.com/5469169/115963121-afaec300-a51e-11eb-9450-ff22a7b39a93.png)
Reconstructed
![image](https://user-images.githubusercontent.com/5469169/115963191-c6edb080-a51e-11eb-9bb7-971262ac725f.png)

**Comment by eugenet12** [comment]
Hi! A few parameters I would look into are:
- Hidden layer dimensions
- Learning rate
- Number of epochs

This task looks more difficult than the MNIST digits (e.g. a "car" has photos from many different angles). Would be interested to see how it turns out!
